hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -2192,8 +2216,12 @@ int QuectelNcpClient::modemHardReset(bool powerOff) {
 
 bool QuectelNcpClient::modemPowerState() const {
     // LOG(TRACE, "BGVINT: %d", hal_gpio_read(BGVINT));
+#if HAL_PLATFORM_CELLULAR_MODEM_VINT_INVERTED
     // NOTE: The BGVINT pin is inverted
     return !hal_gpio_read(BGVINT);
+#else
+    return hal_gpio_read(BGVINT);
```

Comment:
I think this will also break other platforms that use Quectel NCP.

Reply:
I flipped the default for `HAL_PLATFORM_CELLULAR_MODEM_VINT_INVERTED` to be 1, and `HAL_PLATFORM_CELLULAR_MODEM_VOLTAGE_TRANSLATOR` to be 1, and made sure `electron/hal_platform_config.h` contains overrides to 0.